src/dunst.c

```diff
@@ -24,6 +25,34 @@
 
 GMainLoop *mainloop = NULL;
 
+static struct dunst_status status;
+
+/* see dunst.h */
+void dunst_status(const enum dunst_status_field field,
+                  bool value)
```

Comment:
`value` can also be `const`, but is there a point declaring variables on the stack as `const`?

Reply:
"Just because I can":tm:

Actually there is no specific reason. I started doing it with strings (where it's a useful thing).

But yes, either both params should be `const` or not.